WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/Core/TaskManager.cs

```diff
@@ -52,6 +53,50 @@ public partial class DevicePortal
             return processId;
         }
 
+        /// <summary>
+        /// Gets the current running apps on the console based on app processes
+        /// </summary>
+        /// <returns>List of Package Full Names of running apps</returns>
+        public async Task<List<string>> GetRunningAppsAsync()
+        {
+            RunningProcesses appProcesses = await this.GetRunningProcessesAsync();
+            List<string> runningApps = new List<string>();
+
+            foreach (DeviceProcessInfo process in appProcesses.Processes)
+            {
+                // There can be multiple processes per app, so only add an app package name if it doesn't already
+                // exist in the list and it is actually running - processes listed can be in the suspended state
+                if (process.PackageFullName != null && process.IsRunning && !runningApps.Contains(process.PackageFullName))
+                {
+                    runningApps.Add(process.PackageFullName);
+                }
+            }
+
+            return runningApps;
+        }
+
+        /// <summary>
+        /// Gets the current suspended apps on the console based on app processes
+        /// </summary>
+        /// <returns>List of Package Full Names of suspended apps</returns>
+        public async Task<List<string>> GetSuspendedAppsAsync()
+        {
+            RunningProcesses appProcesses = await this.GetRunningProcessesAsync();
+            List<string> suspendedApps = new List<string>();
+
+            foreach (DeviceProcessInfo process in appProcesses.Processes)
+            {
+                // There can be multiple processes per app, so only add an app package name if it doesn't already
+                // exist in the list and it is in a suspended state (not running)
+                if (process.PackageFullName != null && !process.IsRunning && !suspendedApps.Contains(process.PackageFullName))
```

Comment:
I haven't looked at the API's, but just wanted to make sure we could distinguish between suspended and terminated apps.  I guess if the app is terminated it won't even show up on the list?

Reply:
I confirmed once the app is terminated, the processes associated with it do not show up in this list so we can assume they are suspended (unless there is a state I'm forgetting about)